libsel4/sel4_plat_include/bcm2711/sel4/plat/api/constants.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+ * Copyright (C) 2021, Hensoldt Cyber GmbH
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+#pragma once
+
+#ifdef HAVE_AUTOCONF
+#include <autoconf.h>
+#endif
+
+/* Cortex A72 manual, section 10.3 */
+#define seL4_NumHWBreakpoints (10)
```

Comment:
This is wrong, I've just checked the TRM. It should be: 
```
/* Cortex A72 manual, section 10.6.1 */
#define seL4_NumHWBreakpoints (6)
```

See also copmmit in @nspin working repo at https://gitlab.com/arm-research/security/icecap/sel4/-/commit/0cea02f6d1ac6107707972c04c542cf9e0b6ae19#3710b054e41857db7434c68c02adf135d50dd6eb